### ISSUE TITLE
Hydrogen - Integrate into Webpack/Mix

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#1373]
It would smooth out the workflow if the Hydrogen build process could be integrated in the Webpack/Mix bundler.  Then there would be one less step to take to build the projects.  If it could be integrated into the watch/reload mode that would be very convenient as well.

Josh believes it might be possible and is willing to add callbacks if needed.

Would this be helpful or would it just slow the process down for rare changes?

**Comment by esizer** [comment]
Is this helpful? [webpack-shell-plugin](https://www.npmjs.com/package/webpack-shell-plugin).

Maybe:

```js
module.exports = {
  ...
  plugins: [
    new WebpackShellPlugin({onBuildEnd:['npm run h2-build']})
  ],
  ...
}
```

**Comment by JoshBeveridge** [comment]
@esizer @petertgiles This does look promising 😮

**Comment by patcon** [comment]
Maybe helpful resources:
- https://webpack.js.org/plugins/watch-ignore-plugin/
- https://webpack.js.org/api/compiler-hooks/

**Comment by gobyrne** [comment]
related to #3051

**Comment by JoshBeveridge** [comment]
This is resolved and completed in #3051
